deployment/deployments/rskSovrynMainnet/LoanTokenLogicProxy.json

```diff
@@ -0,0 +1,636 @@
+{
+  "abi": [
```

Comment:
1) it looks like there is no way to tell what LoanTokenLogicProxy contract address is used in the tokens,   
pls add a view function getTarget() to the LoanTokenLogicProxy contract to return `_target`
1) python deployment script deploy a separate LoanTokenLogicProxy for each LoanToken while one contract can be used 
pls add deployment script for the LoanTokenLogicProxy and register the same instance in all the tokens - use arrays/maps and cycle through

Reply:
Updated

Reply:
right when speaking about deployment and using current LoanTokenLogicProxy implementation - all you need is interface  
but what i mean - we do need it in case we want to replace LoanTokenLogicProxy and now we have it deployed per contract which is suboptimal and hard to manage and for clarity we do need to tell what LoanTokenLogicProxy contract is used as a proxy for LoanTokens (iTokens) - it should be deployed and verified and deployment and replacement script should be ready and only one contract should be used for all the tokens for now

Reply:
it was decided that this issue to to be done in a separate ticket SOV-2857